README.md

```diff
@@ -49,6 +65,18 @@ is to download a pre-built release binary.
 You can find these binaries on the GitHub
 [releases](https://github.com/brimdata/zed/releases) page.
 
+If you have Go installed, you can easily install `zed` and `zq` by running
+```
+go install github.com/brimdata/zed@v1.0
```

Comment:
1. `go install` needs main packages. We could say `github.com/brimdata/zed/cmd/...` but that will install the dev tools in `cmd`, and I think we should exclude those. While we should eventually move them out of `cmd` or add build tags to them, the easy thing to do now is `github.com/brimdata/zed/cmd/{zed,zq}` (works in any reasonable shell).
2. `latest` selects the highest available released version, and I think that's what we want here. (It doesn't work right now because v0.33.0 still has a replace directive in `go.mod`, but it will work once we tag the next release.)

```suggestion
go install github.com/brimdata/zed/cmd/{zed,zq}@latest
```